### PR TITLE
Fix `test_keeper_force_recovery`

### PR DESCRIPTION
**Pull request by antonio2368**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
This is an automated comment for commit 3815a6aa62414fcd4e51090450a08bc368fd8c86 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/52408/3815a6aa62414fcd4e51090450a08bc368fd8c86/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: tests/integration/test_keeper_force_recovery_single_node/configs/enable_keeper3.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <clickhouse>
     <keeper_server>
+        <use_cluster>false</use_cluster>
```
Comment (alexey-milovidov) [inline tests/integration/test_keeper_force_recovery_single_node/configs/enable_keeper3.xml:3]: Ok. It means, "don't use this config for the client". But I don't understand why this change was needed.

**Review by alexey-milovidov — APPROVED** [review]

File: tests/integration/test_keeper_force_recovery_single_node/configs/enable_keeper3.xml (hunk 1 above)
Comment (antonio2368) [reply tests/integration/test_keeper_force_recovery_single_node/configs/enable_keeper3.xml:3]: To reduce the number of logs we create on Keeper. Recovery has some troubles if there are too many configuration changes and logs to sync on new nodes.
This is hard to solve with current approach but a better and easier way of recovery can now be defined using new `reconfig` (https://github.com/ClickHouse/ClickHouse/pull/49450).
